### PR TITLE
bitlbee-discord: init at 2017-12-27

### PR DESCRIPTION
**Pull request by Lassulus**

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/bitlbee-discord/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  src = fetchFromGitHub {
+    rev = "6a03db169ad44fee55609ecd16e19f3c0f99a182";
+    owner = "sm00th";
+    repo = "bitlbee-discord";
+    sha256 = "1ci9a12c6zg8d6i9f95pq6dal79cp4klmmsyj8ag2gin90kl3x95";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ bitlbee autoconf automake libtool glib ];
```
Comment (joachifm) [inline pkgs/applications/networking/instant-messengers/bitlbee-discord/default.nix]: I expect autoconf, automake, and libtool can be treated as native inputs as well.  You can also try replacing them with `autoreconfHook`, it adds those things & does autogen and whatnot.

File: pkgs/applications/networking/instant-messengers/bitlbee-discord/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{ fetchurl, fetchFromGitHub, stdenv, bitlbee, autoreconfHook, pkgconfig, glib }:
```
Comment (joachifm) [inline pkgs/applications/networking/instant-messengers/bitlbee-discord/default.nix]: `fetchurl` is unused

File: pkgs/applications/networking/instant-messengers/bitlbee-discord/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  nativeBuildInputs = [ autoreconfHook pkgconfig ];
+  buildInputs = [ bitlbee glib ];
+
+  preConfigure = ''
+    export BITLBEE_PLUGINDIR=$out/lib/bitlbee
+    ./autogen.sh
+  '';
+
+  meta = {
```
Comment (joachifm) [inline pkgs/applications/networking/instant-messengers/bitlbee-discord/default.nix]: Please consider adding yourself as the maintainer

**Review by joachifm — COMMENTED** [review]
LGTM. Added two minor nits

File: pkgs/applications/networking/instant-messengers/bitlbee-discord/default.nix (hunk 3 above)
Comment (Lassulus) [reply pkgs/applications/networking/instant-messengers/bitlbee-discord/default.nix]: done

**Comment by Lassulus** [comment]
@joachifm ping?

**Pull request merged**